server/src-lib/Hasura/GraphQL/Context.hs

```diff
@@ -222,6 +227,50 @@ mkCompExpInp colTy =
         )
       ]
 
+    -- Geometry related ops
+    stDWithinOpInpVal =
+      InpValInfo (Just stDWithinDesc) "_st_d_within" $ G.toGT stDWithinInpTy
+    stDWithinDesc =
+      "is the column within in the given distance from given geometry value"
+
+    isGeometryTy = case colTy of
+      PGGeometry -> True
+      _          -> False
+
+    geomOpToInpVal (op, desc) =
+      InpValInfo (Just desc) op $ G.toGT $ mkScalarTy PGGeometry
+    geomOps =
+      [
+        ( "_st_contains"
+        , "does the column contain given geometry value"
+        )
+      ,
+        ( "_st_contains_in"
+        , "is the column contained in given geometry value"
+        )
+      , ( "_st_crosses"
+        , "does the column crosses given geometry value"
+        )
+      , ( "_st_disjoint"
+        , "does the column don't spatially intersect the given geometry value"
```

Comment:
"the opposite of _st_intersects, it is often more efficient to use _not: {_st_intersects: value} over this operator"